package.json

```diff
@@ -263,5 +265,8 @@
     "Jenkinsfile": "lint:jenkinsfile",
     "*package.json": "lint:package-names"
   },
-  "snyk": true
+  "snyk": true,
```

Comment:
Can we disable as this doesn't work on windows ?

Reply:
We can't get rid of it yet since we need it to meet security requirements. Once we update to the npm@>6, we can get rid of it in place of `npm-audit`

Reply:
@bzang are you aware of anything that will fail if we start enforcing npm 6?

Reply:
The only thing that I've been able to find is that the minimum version of node is 6. Currently we supposedly still support back to v4. I think it's reasonable to force developers to move to version of node that's newer.